content/feature/example.md

```diff
@@ -5,12 +5,22 @@ Date: "2015-01-17"
 
 
 ### Description
-My fancy description. Explain what your code does, and what is the purpose of it.
+The code changes the Style properties of the Block and configures it such that Scrollbars will be shown only when overflow is present.
```

Comment:
The code gallery is only meant for SQL commands. This is a bug fix for the website, so don't include information about your bugfix in example.md